airbyte-config/init/src/main/resources/seed/destination_definitions.yaml

```diff
@@ -60,10 +60,6 @@
   dockerImageTag: 1.2.9
   documentationUrl: https://docs.airbyte.com/integrations/destinations/bigquery
   icon: bigquery.svg
-  normalizationConfig:
-    normalizationRepository: airbyte/normalization
-    normalizationTag: 0.2.25
-    normalizationIntegrationType: bigquery
```

Comment:
To confirm: are we saying here that this destination does not support normalization? (from the name `-denormalized` i would assume so, but confirming 😄 )